### ISSUE TITLE
Support arguments in load_plugin()

### ISSUE THREAD

**Issue by stphnlyd** [EntropyOrg/p5-Devel-IPerl#95]
See https://github.com/EntropyOrg/p5-Devel-IPerl/blob/0f9fe2f1470a347705b5ce5bac7fa8fb40904869/lib/IPerl.pm#L40

Now this load_plugin method does not support arguments. It would be best to support arguments like `my ($class, $plugin_name, @args) = @_`, and register plugins by `$plugin_package->register( $class, @args );`, so plugin developer can have arguments to fine control behavior of their plugins.

**Comment by zmughal** [comment]
Yes, both this and a way to expose plugin settings. I need this in `PDLGraphicsGnuplot` as well.
